### ISSUE TITLE
Aggregation functions such as sum(), min(), max() are incorrectly indented if tab_space_size is set to non-default value (i.e. not 4)

### ISSUE THREAD

**Issue by tjirab** [sqlfluff/sqlfluff#1183]
SQL code:
```sql
select a,b
,sum(c) as n
from table.users
group by 1
```

## Expected Behaviour
```sql
select
  a,
  b,
  sum(c) as n
from table.users
group by 1
```

## Observed Behaviour
```sql
select
  a,
  b,
   sum(c) as n
from table.users
group by 1
```

## Steps to Reproduce
- create .sql file
- `sqlfluff fix --force <.sql file>`

Please note that `tab_space_size = 4` yields correct indentation. It seems like `sum()` indentation is not correctly applied. The same applied to `avg()`, `min()`, `max()`

Note: when running `sqlfluff fix <.sql file> again, the issue is recognized:
```bash
L:   4 | P:   4 | L003 | Indentation not consistent with line #3
```

## Version
Include the output of `sqlfluff --version` along with your Python version
```bash
❯ sqlfluff --version
sqlfluff, version 0.6.0
❯ python -V
Python 3.8.5
```

## Configuration
```
[sqlfluff:rules]
tab_space_size = 2
capitalisation_policy = lower
```
Dialect: ansi.

**Comment by tunetheweb** [comment]
Can't repeat. Looks to me like this was fixed with https://github.com/sqlfluff/sqlfluff/commit/baf3c50c428ffc280c9a9ff9429c7c9873aa3a0b.

So think this can be closed.

**Comment by barrywhart** [comment]
Confirmed that this is fixed. If that was the commit that fixed it, that's a cool side effect, since I was working on a different kind of bug. Rules returning "bad" `LintFix` objects is a recurring problem. Someday it'd be cool to have a way to detect and flag these proactively, rather than wait for users to find them.
